src/renderers/contentful/renderDefaultLocale.ts

```diff
@@ -7,5 +8,5 @@ export default function renderDefaultLocale(locales: Locale[]): string {
     throw new Error("Could not find a default locale in Contentful.")
   }
 
-  return `export type CONTENTFUL_DEFAULT_LOCALE_CODE = '${defaultLocale.code}';`
+  return `export type CONTENTFUL_DEFAULT_LOCALE_CODE = '${escapeSingleQuotes(defaultLocale.code)}';`
```

Comment:
Same as above, is there a need to escape the locale code? I'd expect locale codes to be only letters and dashes